utils/frame/benchmarking-cli/src/pallet/template.hbs

```diff
@@ -34,22 +34,22 @@ impl<T: frame_system::Config> {{pallet}}::WeightInfo for WeightInfo<T> {
 		{{~#if (not c.is_used)}}_{{/if}}{{c.name}}: u32, {{/each~}}
 	) -> Weight {
 		// Minimum execution time: {{underscore benchmark.min_execution_time}} nanoseconds.
-		Weight::from_ref_time({{underscore benchmark.base_weight}} as u64)
+		Weight::from_ref_time({{underscore benchmark.base_weight}}_u64)
```

Comment:
```suggestion
		Weight::from_ref_time({{underscore benchmark.base_weight}})
```

The compiler should be able to defer this automatically here and for the other use cases of raw numbers.